index.d.ts

```diff
@@ -2004,7 +2088,7 @@ declare namespace Eris {
   export class ExtendedUser extends User {
     email: string;
     mfaEnabled: boolean;
-    premiumType: 0 | 1 | 2;
+    premiumType: (Constants["PremiumType"])[keyof Constants["PremiumType"]];
```

Comment:
```suggestion
    premiumType: Constants["PremiumType"][keyof Constants["PremiumType"]];
```
Oh man there's more here

Reply:
god damn it bsian